integration_test/ndm_util/ndm_util.go

```diff
@@ -632,3 +633,55 @@ func Clean() {
 		}
 	}
 }
+
+// MatchNDMDeviceList is used to match the NDM devices and device
+// paths specified as string. Both include and exclude can be
+// matched using by changing the pathType bool. pathType `true`
+// means exclude filter will be checked and pathType `false`
+// means include filter will be checked.
+func MatchNDMDeviceList(pathType bool, devicePaths ...string) (bool, error) {
```

Comment:
IMO argument name `pathType` does not make much sense here, because if it is `true` we are checking `exclude filter` and if it is `false` we check `include filter`. We may either
- Use something like `MatchNDMDeviceListOptions` struct in which we may set true to the fields named like `IncludeFilterCheck` and/or `ExcludeFilterCheck` to `true`/`false`. Or,
- To avoid much of the code change, we may just change `pathType` to something like `checkExcludeFilter`, so that if it is `true` we shall check `exclude filter` and if it is `false` we shall check `include filter` instead. Just check it.